docs/pages/development/vc.rst

```diff
@@ -0,0 +1,69 @@
+.. _using-concepts:
+
+Using Concepts & VCs
+####################
+
+Issuing VCs
+***********
+
+As a node, you can issue credentials with each DID you control (If they are trusted is a different story).
+A credential is issued through the API or CLI. The format of the Verifiable Credential is required, not the concept format.
```

Comment:
I don't understand "The format of the Verifiable Credential is required, not the concept format."

Reply:
I'll remove the line since the API spec shows what is expected.